modules/data.atmosphere/R/met.process.R

```diff
@@ -238,15 +250,26 @@ met.process <- function(site, input_met, start_date, end_date, model,
                                           spin = spin)
     
     model.id  <- met2model.result$model.id
-    outfolder <- met2model.result$outfolder
+    model.file.info <- db.query(paste0("SELECT * from dbfiles where id = ", model.id$dbfile.id), con)
+    model.file <- file.path(model.file.info$file_path,model.file.info$file_name)
+    
   } else {
-    model.id  <- ready.id
-    outfolder <- file.path(dir, paste0(met, "_site_", str_ns))
+    PEcAn.utils::logger.info("ready.id",ready.id,machine.host)
+    model.id  <- dbfile.check("Input", ready.id, con)#, hostname=machine.host)
```

Comment:
I left that in as a reminder that I still need to get this working for the remote case